app/controllers/api/v2/admin_controller.rb

```diff
@@ -44,11 +44,16 @@ def search_users
     @username = data.fetch('username', '')
     @email = data.fetch('email', '')
 
-    @users, @paginate = paginate User.where(
-      'UPPER(username) = UPPER(?) or UPPER(email) = UPPER(?)',
-      @username.to_s,
-      @email.to_s
-    )
+    @query = User.all
+    if @username.present?
+        @query = @query.where('UPPER(username) = UPPER(?)', @username.to_s)
```

Comment:
```suggestion
      @query = @query.where('UPPER(username) = UPPER(?)', @username.to_s)
```

Reply:
Using 2 spaces instead of 4

Reply:
Ah ops, didn't notice this. I'll update

FYI I usually add this file https://editorconfig.org/ to projects, so I don't have to think about this when switching projects